website/docs/r/maintenance_window.html.markdown

```diff
@@ -8,7 +8,9 @@ description: |-
 
 # Resource: mongodbatlas_maintenance_window
 
-`mongodbatlas_maintenance_window` provides a resource to schedule a maintenance window for your MongoDB Atlas Project and/or set to defer a scheduled maintenance up to two times.
+`mongodbatlas_maintenance_window` provides a resource to schedule the maintenance window for your MongoDB Atlas Project and/or set to defer a scheduled maintenance up to two times. Please refer to [Maintenance Windows](https://www.mongodb.com/docs/atlas/tutorial/cluster-maintenance-window/#configure-maintenance-window) documentation for more details.
+
+-> **NOTE:** Only a single maintenance window resource can be defined per project.
```

Comment:
knit: i think we normally have the groupId note first but I think it's better in this way